recipe/meta.yaml

```diff
@@ -1,51 +1,49 @@
 {% set name = "numba" %}
-{% set version = "0.60.0" %}
+{% set version = "0.61.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5df6158e5584eece5fc83294b949fd30b9f1125df7708862205217e068aabf16
+  sha256: 888d2e89b8160899e19591467e8fdd4970e07606e1fbc248f239c89818d5f925
   patches:
-    # This forces a benign DeprecationWarning in test_import.py to be ignored.
-    - patches/ignore_deprecation_warning_in_import_test.patch  # [py>=310 and win]
+    - 0001-skip-pycc-and-overlaod_allocation-tests.patch
 
 build:
-  number: 1
+  number: 0
   skip: true  # [python_impl == 'pypy']
-  skip: true  # [(py<39 or py>=313) or s390x]
+  skip: true  # [(py<310 or py>=314) or s390x]
   script:
     - export CC="${CC} -pthread"  # [linux]
     - export CXX="${CXX} -pthread"  # [linux]
     - {{ PYTHON }} -m pip install . --no-deps  --no-build-isolation --ignore-installed -vv
   entry_points:
     - numba = numba.misc.numba_entry:main
-  ignore_run_exports:
-    - libllvm14 >=14.0.6,<14.1.0a0
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
-    - m2-patch  # [win and py>=310]
+    - m2-patch  # [win]
+    - patch     # [unix]
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.43.0
-    - numpy 2.0.2
+    - llvmlite 0.44.0
+    - numpy 2.1.3
```

Comment:
Isn't `2.1` more flexible for future rebuilds?

Reply:
That should be a problem. Unless we add a new subdir.